control-plane/config/post-install/500-storage-version-migrator.yaml

```
@@ -0,0 +1,41 @@
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: knative-kafka-storage-version-migrator-
```

We're using name for the other one so that we can apply it.

https://github.com/knative-sandbox/eventing-kafka-broker/blob/976a0dd1ab2a877217b0d8c2ed914752767d8464/control-plane/config/post-install/500-post-install-job.yaml#L18

https://github.com/knative-sandbox/eventing-kafka-broker/blob/976a0dd1ab2a877217b0d8c2ed914752767d8464/test/e2e-common.sh#L144

can we use the name here too?

```suggestion
  name: knative-kafka-storage-version-migrator
```

---

eventing uses `generatedName` (does require `kubectl create`).

---

I am fine w/ either way :-) 

what's really against the generated name? (except for the k `create` vs `apply`)

---

- create vs apply
- consistency with the other job